src/tos/helpers/getTransComponents.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react';
+
+export default function getTransComponents(links: Array<string> | undefined) {
+  if (links === undefined) return [];
+  const components: Array<ReactNode> = [];
+
+  links.forEach(link => {
```

Comment:
It would be nicer to use `links.map(...)` instead of forEach.

Reply:
Changed to .map